config/application.config.php

```diff
@@ -1,28 +1,28 @@
 <?php
 /**
  * @license   http://opensource.org/licenses/BSD-3-Clause BSD-3-Clause
- * @copyright Copyright (c) 2014 Zend Technologies USA Inc. (http://www.zend.com)
+ * @copyright Copyright (c) 2014-2016 Zend Technologies USA Inc. (http://www.zend.com)
  */
 
-return array(
+return [
     // Retrieve the list of modules for this application.
     'modules' => include __DIR__ . '/modules.config.php',
     // This should be an array of paths in which modules reside.
     // If a string key is provided, the listener will consider that a module
     // namespace, the value of that key the specific path to that module's
     // Module class.
-    'module_listener_options' => array(
-        'module_paths' => array(
+    'module_listener_options' => [
+        'module_paths' => [
             './module',
             './vendor'
-        ),
+        ],
         // Using __DIR__ to ensure cross-platform compatibility. Some platforms --
         // e.g., IBM i -- have problems with globs that are not qualified.
-        'config_glob_paths' => array(realpath(__DIR__) . '/autoload/{,*.}{global,local}.php'),
+        'config_glob_paths' => [realpath(__DIR__) . '/autoload/{,*.}{global,local}.php'],
```

Comment:
WRONG! :smile:

IBM i Series users have reported to us in the past that globbing, even when using zend-stdlib, fails unless the real path is used. As such, we provide it both in here and the ZF skeleton application.